src/run/handlers/server.ts

```diff
@@ -138,6 +169,20 @@ export default async (request: Request) => {
       await waitForBackgroundWork()
     }
 
+    topLevelSpan.setAttributes({
+      isBackgroundRevalidation: requestContext.isBackgroundRevalidation,
+      ...expandHeaders('request.headers', request.headers),
+      ...expandHeaders('response.headers', response.headers),
+    })
+
+    // requestContext.logger
+    //   .withFields({
+    //     isBackgroundRevalidation: requestContext.isBackgroundRevalidation,
+    //     ...expandHeaders('request.headers', request.headers),
+    //     ...expandHeaders('response.headers', response.headers),
+    //   })
+    //   .debug('[NetlifyNextRuntime] Response cache headers')
```

Comment:
I typically don't like keeping comments like this around. My preference is to wrap this in a debug flag (if possible) and when set to true it logs the line, or remove the comment

Reply:
Ops, those were not meant to be commited - I will remove them next time I have a chance

Reply:
Ammended last commit without unintended commented out code - https://github.com/opennextjs/opennextjs-netlify/pull/2765/commits/45f3a0a51d32ad7a42ae0b4a21b81cef46cf35a8